packages/plugin/webpack/src/util/rendererTypeUtils.ts

```diff
@@ -36,7 +40,7 @@ export const isNoWindowEntries = (entries: WebpackPluginEntryPoint[]): entries i
 
 export const isPreloadOnlyEntries = (entries: WebpackPluginEntryPoint[]): entries is WebpackPluginEntryPointPreloadOnly[] => {
   for (const entry of entries) {
-    if (!isPreloadOnly(entry)) {
+    if (!hasPreloadScript(entry)) {
```

Comment:
changed from previous iteration's `isPreloadOnly` since now, an entry can have `html` + `preload.js` in this bucket